src/SpeedInput/_tests_/SpeedInput.test.jsx

```diff
@@ -0,0 +1,104 @@
+import { render, fireEvent, cleanup } from '@testing-library/react';
+import SpeedInput, { SPEED_UNITS, Mode } from '../SpeedInput';
+
+afterAll(cleanup);
+
+const options = [
+    {
+        label: 'Min',
+        value: 1
+    },
+    {
+        label: 'max',
+        value: 50
+    },
+    {
+        label: 'unit',
+        value: 'Mph'
+    },
+    {
+        label: 'unit',
+        value: 'Kph'
+    },
+    {
+        label: 'unit',
+        value: 'm/s²'
+    },
+];
+
+const vehicleValue = {
+    min: 1,
+    max: 50,
+    unit: SPEED_UNITS.kilometersPerHour,
+    mode: Mode.minmax
+};
+
+describe('<SpeedInput>', () => {
+
+    it('Should exist', () => {
+        const callback = jest.fn();
+        const speedInput = render(
+            <SpeedInput
+                value={vehicleValue}
+                unitConversionRequired={true}
+                allowNegativeInput={true}
+                disabled={false}
+                defaultUnits={SPEED_UNITS.kilometersPerHour}
+                onChange={callback}
+            />);
+        expect(speedInput).toBeDefined();
+    });
+
+    it('renders disabled', () => {
+        const callback = jest.fn();
+        const { container } = render(
+            <SpeedInput
+                value={vehicleValue}
+                unitConversionRequired={true}
+                allowNegativeInput={true}
+                disabled={true}
+                defaultUnits={SPEED_UNITS.kilometersPerHour}
+                onChange={callback} />
+        );
+        const speedInputContainer = container.querySelectorAll('div');
+        expect(speedInputContainer).toBeDisabled;
+    });
+
+    it('Takes in all props', () => {
+        const callback = jest.fn();
+        const { container, getByText } = render(
+            <SpeedInput
+                value={vehicleValue}
+                unitConversionRequired={true}
+                allowNegativeInput={false}
+                disabled={false}
+                defaultUnits={SPEED_UNITS.kilometersPerHour}
+                onChange={callback}
+            />
+        );
+        expect(container).toBeInTheDocument;
```

Comment:
toBeInTheDocument is a function. Please replace it with this expect(container).toBeInTheDocument();